plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/marker/resolution/FordiacMarkerResolutionGenerator.java

```diff
@@ -29,10 +32,15 @@ public IMarkerResolution[] getResolutions(final IMarker marker) {
 		return switch (FordiacErrorMarker.getCode(marker)) {
 		case LibraryElementValidator.ITYPED_ELEMENT__VALIDATE_TYPE,
 				LibraryElementValidator.CONFIGURABLE_FB__VALIDATE_DATA_TYPE ->
-			new IMarkerResolution[] { new ChangeDataTypeMarkerResolution(marker),
-					new CreateDataTypeMarkerResolution(marker) };
-		case LibraryElementValidator.TYPED_CONFIGUREABLE_OBJECT__VALIDATE_TYPE -> new IMarkerResolution[] {
-				new CreateMissingFBMarkerResolution(marker), new ChangeFBMarkerResolution(marker) };
+			Stream.concat(
+					Arrays.stream(new IMarkerResolution[] { new CreateDataTypeMarkerResolution(marker),
```

Comment:
I suggest using `Stream.of` instead of wrapping an array in a stream.

Reply:
had not thought of that - thanks